<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserData</class>
 <widget class="QDialog" name="UserData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_zip">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>City:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_birthdate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ZIP Code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_city">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Telephone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_telephone">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_state">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>State:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hey, dude!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Be careful! This is sensitive data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>600</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_picture">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserData</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
